apps/web/app/api/defaultResponderForAppDir.ts

```diff
@@ -21,10 +19,15 @@ export const defaultResponderForAppDir = <T extends NextResponse | Response = Ne
     let ok = false;
     try {
       performance.mark("Start");
-
-      const result = endpointRoute
-        ? await wrapApiHandlerWithSentry(async () => await handler(req, { params }), endpointRoute)()
-        : await handler(req, { params });
+      let result: T | undefined;
+      if (process.env.NODE_ENV === "development") {
```

Comment:
It's strange to have the same line of code on 24 and 29. 

If we make this if as follows and then just an `else` afterwards, I think we still get the same benefit of the dynamic import, right?

```suggestion
      if (process.env.NODE_ENV === "development" || !endpointRoute) {
```